src/arduino/respi-prod.cpp

```diff
@@ -22,11 +22,13 @@
 // External
 #include <AnalogButtons.h>
 #include <Arduino.h>
+#include <HardwareTimer.h>
```

Comment:
inutile car HardwareTimer.h deja inclus dans air_transistor.h

Reply:
And mainly included by Arduino.h